### PR TITLE
Add apache-tika quickstart

### PR DESCRIPTION
**Pull request by sberyozkin**
This PR depends on https://github.com/quarkusio/quarkus/pull/4360 because it uses a new `odf` (OpenDocumentFormat) abbreviation

### CODE REVIEW

File: using-apache-tika/pom.xml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        </dependency>
+        <dependency>
+            <groupId>io.rest-assured</groupId>
+            <artifactId>rest-assured</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+     <plugins>
```
Comment (gsmet) [inline using-apache-tika/pom.xml]: The indentation is broken here. If you can reindent the file with 4 spaces, it's better.

**Comment by sberyozkin** [comment]
@gsmet Renamed and fixed the formatting; but not sure how to change the commit message as the `r` instruction did not work.

**Comment by gsmet** [comment]
@sberyozkin for the last commit, you can just use `git commit --amend`. That's what I did before merging.

Thanks!

**Comment by sberyozkin** [comment]
@gsmet, thanks, did not realize I could follow up with this command after a rebase :-)

**Pull request merged**